inc/authldap.class.php

```diff
@@ -3945,4 +4007,33 @@ public static function getUsers($values, &$results, &$limitexceeded) {
 
       return $users;
    }
+
+   public function checkFilesExist(&$input): bool {
```

Comment:
Reference flag does not seems usefull.
```suggestion
   public function checkFilesExist(array $input): bool {
```